PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,41 +1,26 @@
 <!--
 Thanks for your contribution!
```

Comment:
Why are we treating the phrase 'pull request' as some sort of a proper noun or something? Why capitalize it and wrap it in backticks?